spec.bs

```diff
@@ -2862,17 +2862,53 @@ partial interface Navigator {
   Promise<AdAuctionData> getInterestGroupAdAuctionData(AdAuctionDataConfig config);
 };
 
+dictionary AdAuctionData {
+  required Uint8Array request;
+  required USVString requestId;
+};
+</xmp>
+
+<xmp class="idl">
 dictionary AdAuctionDataConfig {
   required USVString seller;
   required USVString coordinatorOrigin;
+  unsigned long requestSize;
+  record<USVString, AdAuctionDataBuyerConfig> perBuyerConfig;
 };
+</xmp>
 
-dictionary AdAuctionData {
-  required Uint8Array request;
-  required USVString requestId;
+<dl class=domintro>
+  <dt>{{AdAuctionDataConfig/seller}}
+  <dd>The seller that will be used as the {{AuctionAdConfig/seller}} in the
+    following {{AuctionAdConfig}} passed to {{Window/navigator}}.{{Navigator/runAdAuction()}}.
+  <dt>{{AdAuctionDataConfig/coordinatorOrigin}}
+  <dd>The origin of the coordinator hosting public encryption keys for the server
+    running the ad auction. The [=origin/scheme=] must be "`https`". An implementation
+    may select which coordinators are acceptable.

```

Comment:
What happens when the implementation deems a coordinator unacceptable?

Reply:
That's in https://github.com/brusshamilton/turtledove/pull/2. We reject the promise with a {{TypeError}}.